mods/player_api/README.txt

```diff
@@ -30,3 +30,6 @@ celeron55, Perttu Ahola <celeron55@gmail.com> (CC BY-SA 3.0):
 sonictechtonic (CC BY 3.0):
 https://www.freesound.org/people/sonictechtonic/sounds/241872/
   player_damage.ogg
+
+An0n3m0us (CC BY 3.0):
+  character.blend -- Player animation improvement
```

Comment:
`character.b3d` too as you have altered that?

Reply:
Oh yeah, of course xD . I'll do that too.

Reply:
Done.